.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -48,6 +48,7 @@ body:
     - SDK
     - Wing Console
     - Playground
+    - Tutorial
```

Comment:
Can you please also update the contributor's guide?
https://github.com/winglang/wing/blob/main/docs/06-contributors/030-pull-requests.md

Reply:
Done

Reply:
Sorry @ShaiBer, forgot this one too - could you update as well please?
https://github.com/winglang/wing/blob/main/.github/workflows/pull-request-lint.yml